source/monitoring/cluster-monitoring.txt

```diff
@@ -0,0 +1,130 @@
+.. _c-cluster-monitoring:
+
+==================
+Cluster Monitoring
+==================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, server, topology
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecols
+
+Overview
+--------
+
+This guide shows you how to use the {+driver-short+} to monitor server
+discovery and monitoring (SDAM) events in a MongoDB instance, replica
+set, or sharded cluster. These events occur when there are any changes
+in the state of the MongoDB instance or cluster that you are connected
+to.
+
+You might use information about SDAM events in your application to
+understand cluster changes, assess cluster health, or perform capacity
+planning.
+
+.. _c-subscribe-sdam:
+
+Subscribe to Events
+-------------------
+
+You can access details about SDAM events by subscribing to them
+in your application. To subscribe to an event, create a value of
+type ``mongoc_apm_callbacks_t`` and create an event callback function
+for the event you want to subscribe to. Use the callback's corresponding
+setter method to set the callback on the ``mongoc_apm_callbacks_t`` and 
+pass this type to the ``mongoc_client_set_apm_callbacks()`` function.
+
+This code monitors server opening events by performing the following
+actions:
+
+- Defines a ``server_opening()`` event handler function, or callback
+- Creates a ``mongoc_apm_callbacks_t`` structure to store callbacks
+- Calls the ``mongoc_apm_set_server_opening_cb()`` function, which instructs
+  the ``mongoc_apm_callbacks_t`` type to store a pointer to the ``server_opening()``
+  callback
+- Calls the ``mongoc_client_set_apm_callbacks()`` function, which registers
+  the callback with the client
```

Comment:
```suggestion
  the list of callbacks in the ``mongoc_apm_callbacks_t`` object with the client
```

Reply:
There's only one callback in this code example (server_opening()) rather than a list so I reworded slightly